src/dataMap/metaManager/metaSchema.js

```diff
@@ -299,8 +301,8 @@ export default () => {
      * // as a string, for each column.
      * colWidths: '100px',
      *
-     * // as an array, based on visual indexes. The rest of the columns have a default width.
-     * colWidths: [100, 120, 90],
+     * // as an array, based on visual indexes. Unspecified columns have a default width.
```

Comment:
It would also be nice to link to the `default width` so that a developer would know what to expect. It seems to be 50 in the code, should we explicit that? Is the default width mentioned elsewhere?

Reply:
I mentioned this here: https://github.com/handsontable/handsontable/pull/7524/files#diff-8bc2a244e114493872e4c0dd418a668d1f733a02ba45af2e9983ae8c1fac7b01